basilisk/__main__.py

```
@@ -96,28 +68,33 @@ def parse_args():
 	return parser.parse_args()
 
 
+def action_on_already_running():
+	"""Handle actions when the Basilisk application is already running.
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)